documentation/milestones.md

```diff
@@ -0,0 +1,26 @@
+# Private Chef - Milestones
+
+[M0] Milestone0: **Modelado del problema**
+
++ **Objetivo**: Realizar un análisis de las Historias de Usuario (HU) para definir los elementos clave de las HU.
++ **Entregable**: Código que modele estos elementos del dominio.
++ **Viabilidad**: Cuando sea validado por el revisor o profesor.
```

Comment:
Como os he comentado en clase, esto no dice nada sobre la viabilidad, simplemente traslada a otra persona el decidir si es viable o no. ¿Qué pasa si el revisor o profesor dice que no son viables? ¿Cómo puedes ponerte a trabajar de nuevo?
Siempre tenéis que tratar de incluir, si no criterios puramente objetivos, criterios que permitan justificar el que se considere válido o no. En este caso, ¿qué considerarías tú que es un modelo del problema válido?

Reply:
Cierto, el objetivo de este primer hito (milestone 0) es que un compañero al que se le asigne tu proyecto pueda obtener el apto finalizando este hito. Por lo tanto, el objetivo es que se pueda definir que metodología va a seguir el proyecto para el modelado del problema, pero para eso, deberías de establecer unas directrices o pautas de como o con qué tipo de metodología quieres que siga el proyecto. Recuerda que, tendrás que validárselo tú y continuar con lo que te entregue durante el resto de objetivos, de ahí su importancia.